### PR TITLE
Need MappedArrays: also mappedarray arguments need updating

### PR DESCRIPTION
**Pull request by PetrKryslUCSD**

### CODE REVIEW

**Comment by Tokazama** [comment]
At this point I think it would be worth moving forward with some of these PRs. I'd feel more comfortable if we could get the tests passing though.

**Comment by PetrKryslUCSD** [comment]
Right you are. It was a problem with the Travis configuration. Fixed now.

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name = "NIfTI"
+uuid = "f897abbf-c5d9-45cb-bf89-9f7f579a29ec"
+authors = ["Petr Krysl <pkrysl@ucsd.edu>"]
```
Comment (Tokazama) [inline Project.toml]: Could we delete this line? I don't know if we should or shouldn't have distinct authorship for this package. That should probably be addressed at another time.

Comment (PetrKryslUCSD) [reply Project.toml]: Absolutely no problem. It was generated automatically.

**Comment by Tokazama** [comment]
Thanks for putting this together!

**Pull request merged**